Resources/EditorData/AtomicEditor/editor/ui/renameasset.tb.txt

```diff
@@ -6,10 +6,9 @@ TBLayout: axis: y, distribution: gravity, position: left
 	TBSeparator: gravity: left right, skin: AESeparator
 	TBLayout: gravity: left right, distribution-position: right bottom
 		TBTextField: text: "New Name:"
-		TBEditField: id: new_name, autofocus: 1
+		TBEditField: id: new_name
```

Comment:
Yeah, let's do that since we'll be backing these out once we have a fix for the auto-focus widget event